Documentation/Books/Manual/DataModeling/Documents/DocumentMethods.md

```diff
@@ -1,5 +1,6 @@
 Collection Methods
 ==================
+<!-- the filename is a fail - can it be moved? -->
```

Comment:
There are methods to work with documents on the database level and on the collection level.
"Collection Methods" here means "these are the functions to work with documents via the collection object".

Can we remove this comment please?

Reply:
There is no section "Document Methods" within the file. All calls are methods of collections even though they modify / create / delete documents. I think it would be better to rename the file so it matches it's content. But OK.